framework/platform/bpl/linux/bpl_arp.cpp

```diff
@@ -17,7 +17,7 @@
 
 using namespace beerocks::bpl;
 
-int bpl_arp_mon_start(BPL_ARP_MON_CTX *ctx, const char *iface) { return 0; }
+int bpl_arp_mon_start(BPL_ARP_MON_CTX *ctx, const char *iface) { return -2; }
```

Comment:
Why don't we use `-BPL_ERR_OPERATION_NOT_SUPPORTED` here? (Not actually a change for this PR because we use int instead of enum everywhere, so fixing that would be a separate PR.)

Reply:
We can do this in a followup PR indeed.